apps/api/tests/api/application-state.test.ts

```diff
@@ -129,14 +151,29 @@ describe('State Machine', () => {
 		});
 
 		it('should change from INSTITUTIONAL_REP_REVIEW to CLOSED on close', async () => {
-			const result = await createApplicationStateManager({ id: 2 });
+			const result = await createApplicationStateManager({ id: 3 });
```

Comment:
Changes to applicationId are to break dependencies between tests
Test Utils now creates 20 records (arbitrary) so there are plenty of IDs to work with for any test suite